source/lex.h

```diff
@@ -100,11 +100,11 @@ enum class lexeme : std::int8_t {
 
 auto is_literal(lexeme l) {
     switch (l) {
-    break;case lexeme::FloatLiteral:        
-          case lexeme::BinaryLiteral:       
-          case lexeme::DecimalLiteral:      
-          case lexeme::HexadecimalLiteral:  
-          case lexeme::StringLiteral:       
+    break;case lexeme::FloatLiteral:
+          case lexeme::BinaryLiteral:
+          case lexeme::DecimalLiteral:
+          case lexeme::HexadecimalLiteral:
+          case lexeme::StringLiteral:
```

Comment:
My editor trims white spaces at the end of lines automatically, I can get rid of those diffs if they are an issue. I'm usually not a fan of "whitespace" only diffs...